script.js

```diff
@@ -0,0 +1,49 @@
+function addZeroes(num) {
+    const stringNum = num.toString().split('.')[1]
+    const len = stringNum && stringNum.length > 2 ? stringNum.length : 2
+    return Number(num).toFixed(len)
+}
+
+$('#target').submit(function(e){
+    e.preventDefault()
+    const bill = $('.money').val()
+    const tip = $('.radio:checked').val()
+    const tipPercentage = (tip / 100) + 1
+    const totalAmountUnrounded = bill * tipPercentage
+        const totalAmountRounded = Math.round(totalAmountUnrounded * 100) / 100
+    const tipAmountUnrounded = totalAmountRounded - bill
+        const tipAmountRounded = Math.round(tipAmountUnrounded * 100) / 100
+
+        let finalTip = addZeroes(tipAmountRounded)
+        let finalBill = addZeroes(totalAmountRounded)
+
+    if(isNaN(bill) === true) {
```

Comment:
It's a good practice to avoid testing for number with `isNaN` when you try to use it on a value which is not a floating-point number, Javascript first tries to convert that to a number and then checks if the number has the special NaN value. To make matters more confusing, `undefined` is also treated as `NaN` in JavaScript, so when the value can not be converted to a number, it is considered NaN.

In this case, it's better to use `typeof bill !== 'number'`